build/lib/test/test.js

```diff
@@ -712,11 +720,11 @@
 					adbTargetArgs = '-d';
 					// FIXME: Grab device listing and pick first one?!
 				}
-				await exec(`adb ${adbTargetArgs} shell "run-as ${APP_ID} cat '${filepath}'" > ${dest}`);
+				await exec(`${adbPath} ${adbTargetArgs} shell "run-as ${APP_ID} cat '${filepath}'" > ${dest}`);
 			} else {
-				// await exec(`adb -e shell "run-as ${APP_ID} cat '${filepath}'" > ${dest}`);
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 885;
-->

- :warning: build/lib/test/test.js line 885 – Missing JSDoc parameter description for 'testResults'. (valid-jsdoc)